pkgs/top-level/all-packages.nix

```diff
@@ -25389,6 +25389,8 @@ with pkgs;
   stlink = callPackage ../development/tools/misc/stlink { };
   stlink-gui = callPackage ../development/tools/misc/stlink { withGUI = true; };
 
+  stlink-tool = callPackage ../development/tools/misc/stlink-tool { };
```

Comment:
Please move the package to `pkgs/by-name/st/stlink-tool/package.nix` and drop this line, thanks.

Reply:
Done